### PR TITLE
fix: updates line-height

### PR DESCRIPTION
**Pull request by adamraider**

### CODE REVIEW

File: packages/core/src/global/_variables.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@ $ray-font-stack-sans-serif: 'Apercu', Avenir, -apple-system, BlinkMacSystemFont,
 $ray-font-stack-mono: 'Apercu Mono', SFMono-Regular, Menlo, Monaco, Consolas,
   'Liberation Mono', 'Courier New', monospace;
 $ray-base-font-size: rem(16px);
-$ray-base-line-height: 1.75rem;
+$ray-base-line-height: 1.618;
```
Comment (alanfriedman) [inline packages/core/src/global/_variables.scss]: Why change the global line height?

Comment (adamraider) [reply packages/core/src/global/_variables.scss]: it's necessary

Comment (adamraider) [reply packages/core/src/global/_variables.scss]: The global line height should be relative to the element not the base font-size

Comment (alanfriedman) [reply packages/core/src/global/_variables.scss]: So this is an unrelated fix?

Comment (adamraider) [reply packages/core/src/global/_variables.scss]: it's somewhat related

**Pull request merged**